diff --git a/emscripten/tools/shared.orig.py b/emscripten/tools/shared.py
index 22858bd..0eaff95 100644
--- a/emscripten/tools/shared.orig.py
+++ b/emscripten/tools/shared.py
@@ -449,14 +449,14 @@ def check_sanity(force=False):
     if sanity_is_correct():
       return
 
-    if os.path.exists(sanity_file):
-      sanity_data = utils.read_file(sanity_file)
-      logger.info('old sanity: %s' % sanity_data)
-      logger.info('new sanity: %s' % expected)
-      logger.info('(Emscripten: config changed, clearing cache)')
-      Cache.erase()
-    else:
-      logger.debug(f'sanity file not found: {sanity_file}')
+    # if os.path.exists(sanity_file):
+    #   sanity_data = utils.read_file(sanity_file)
+    #   logger.info('old sanity: %s' % sanity_data)
+    #   logger.info('new sanity: %s' % expected)
+    #   logger.info('(Emscripten: config changed, clearing cache)')
+    #   Cache.erase()
+    # else:
+    #   logger.debug(f'sanity file not found: {sanity_file}')
 
     perform_sanity_checks()
 
diff --git a/emscripten/src/proxyClient.js b/emscripten/src/proxyClient.js
index 6402ae62a..5b622e7dc 100644
--- a/emscripten/src/proxyClient.js
+++ b/emscripten/src/proxyClient.js
@@ -16,13 +16,13 @@ if (typeof Module == 'undefined') {
   };
 }
 
-if (!Object.hasOwnProperty(Module, 'print')) {
+if (!Module.hasOwnProperty('print')) {
   Module['print'] = function(x) {
     console.log(x);
   };
 }
 
-if (!Object.hasOwnProperty(Module, 'printErr')) {
+if (!Module.hasOwnProperty('printErr')) {
   Module['printErr'] = function(x) {
     console.error(x);
   };

