diff --git a/c/tools/brotli.c b/c/tools/brotli.c
index 049371c..dbc80ec 100644
--- a/c/tools/brotli.c
+++ b/c/tools/brotli.c
@@ -1190,9 +1190,9 @@ int main(int argc, char** argv) {
       PrintVersion();
       break;
 
-    case COMMAND_COMPRESS:
-      is_ok = CompressFiles(&context);
-      break;
+    // case COMMAND_COMPRESS:
+    //   is_ok = CompressFiles(&context);
+    //   break;
 
     case COMMAND_DECOMPRESS:
     case COMMAND_TEST_INTEGRITY:
